examples/tcp-close/server.rs

```diff
@@ -293,7 +295,23 @@ impl TcpServer {
 
     /// Issues a close() operation.
     fn issue_close(&mut self, qd: QDesc) -> Result<()> {
-        Ok(self.libos.close(qd)?)
+        if self.should_async_close {
+            let qt: QToken = self.libos.async_close(qd)?;
+
+            match self.libos.wait(qt, None) {
+                Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_CLOSE && qr.qr_ret == 0 => {
+                    println!("async_close_succeeded");
+                    Ok(())
+                },
+                Ok(_) => anyhow::bail!("wait() should succeed with async_close()"),
+                Err(_) => anyhow::bail!("wait() should succeed with async_close()"),
```

Comment:
Isn't it valid that we fail with `ECONNRESET`?

Reply:
Done, handled ECONNRESET.